packages/components/src/components/wallet-cfd-success-dialog/wallet-cfd-success-dialog.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Button from '../button';
+import Modal from '../modal';
+import Text from '../text';
+import WalletCFDCard from './wallet-cfd-card';
+import { isMobile } from '@deriv/shared';
+import { localize } from '@deriv/translations';
```

Comment:
@aizad-deriv I'm not sure about using localize on components package. As I know we can't use it here